css/gloss.css

```diff
@@ -471,4 +474,20 @@ line.selected.just {
 
 .addWitnessBtn{
   margin-bottom: 1em;
+}
+
+form#ngForm
```

Comment:
Using IDs in a general CSS file should only be done if it really needs to be specific to that one element only.

Reply:
As Patrick said this class will only apply to this one specific form.  If you want this to happen on other forms be sure to make it more generic.